heareval/tasks/coughvid.py

```diff
@@ -56,7 +56,8 @@ def run(self):
             "https://zenodo.org/record/4498364/files/public_dataset.zip",
             os.path.join(self.workdir, "corpus.zip"),
         )
-        self.output().open("w")
+        with self.output().open("w") as _:
+            pass
```

Comment:
This is an option too https://stackoverflow.com/questions/1158076/implement-touch-using-python

Reply:
self.output() is a LocalTarget, so you would have to grab the filename and then touch it.